### ISSUE TITLE
Spawn mobs

### ISSUE THREAD

**Issue by nicholas-eden** [OpenDiablo2/OpenDiablo2#432]
On region load, sets of mobs should be placed in the map.  This should read from some data file to identify which mobs should be spawned.  They should be spawned in clusters.

Keep in mind the end goals:
- They will need to move/aggro together, maybe store the clusters as a collection
- They will need to move within an area in path that makes sense (don't get stuck in a corner)

**Comment by nicholas-eden** [comment]
I'm going to try to work on this, without interfering with the server build-up too much.

**Comment by tjarratt** [comment]
Is it also worth calling out that different mobs have different behaviors (and some at different times, depending on conditions)

* those small spell casters in act 5 will sometimes run to corners
* zealots in act 3 will run to engage you, but run away if you kill their healer
* fallen will run to engage you but run away briefly if one of their camarade dies

**Comment by essial** [comment]
levels.txt will be involved in this for sure.

**Comment by nicholas-eden** [comment]
For this first step I'm just going to try to put them on the map, no movement or behaviors.  The AI stuff is going to be its own beast.

**Comment by essial** [comment]
Yeah, and honestly the networking logic is going to cause all the mob spawn stuff to be turned into network packets anyway.
